examples/Validate/check_nodeport.yaml

```diff
@@ -6,10 +6,10 @@ spec:
   rules:
   - name: check-host-path
     resource:
-      kinds: 
+      kinds:
       - Service
     validate:
-      message: "Node port services are not allowed"
+      message: "Only NodePort type is allowed"
       pattern:
         spec:
-          type: "!NodePort"
\ No newline at end of file
+          type: "NodePort"
```

Comment:
This changes / reverses the intent of the policy. Is the ! operator not supported?

Reply:
Operators for the string have not been implemented yet. (https://github.com/nirmata/kyverno/issues/61)